src/main/resources/resources/strings/client/data_de.properties

```diff
@@ -0,0 +1,806 @@
+# JSettlers i18n localization strings: At Client.
+#  This file is part of Java Settlers - An online multiplayer version of the game Settlers of Catan
+#
+# This file Copyright (C) 2013 Luis A. Ramirez <lartkma@gmail.com>
```

Comment:
Can you please update this line to give yourself a quick attribution? An alias or nickname is fine if you'd prefer that. The email part is also optional. Year should be 2023 :)